packages/jest-runtime/src/index.ts

```diff
@@ -568,7 +580,7 @@ export default class Runtime {
 
       const mime = match.groups.mime;
       if (mime === 'application/wasm') {
-        throw new Error('WASM is currently not supported');
+        throw new Error('WASM in data URLs is currently not supported');
```

Comment:
should be somewhat straightforward to support this as well, I think? Just have `_importWasmModule` take some source instead of reading from disk

Reply:
Nice idea, that’s done 🕺